.github/scripts/build_info.py

```diff
@@ -151,39 +153,43 @@ def get_changed_python_dependencies_files() -> List[str]:
     return changed_dependencies_files
 
 
-def should_test_all_python_versions() -> bool:
+def check_if_pr_has_label(action: str, label: str):
+    """
+    Checks if the PR has the given label.
+
+    The function works for all GitHub events, but then returns false.
```

Comment:
nit: would be more clear to say "but returns false for any event that is not a PR"